.circleci/config.yml

```diff
@@ -0,0 +1,274 @@
+version: 2.1
+
+orbs:
+  aws-s3: circleci/aws-s3@1.0.11
+
+jobs:
+  grafana-aws-metrics-master:
+    docker:
```

Comment:
remove this part that is the same for all jobs and do this

in the top of the file
```
executors:
  default:
    docker:
    - image: circleci/golang:1.14.1
    working_directory: /go/src/github.com/mattermost/mattermost-cloud-monitoring
```

for each job that uses that image
```
grafana-aws-metrics-master:
   executor:
       name: default
...
```

Reply:
Done, together with some wider refactoring.